scala-vector-tile/src/main/scala/geotrellis/vectortile/Commands.scala

```diff
@@ -29,11 +28,30 @@ object Command {
 
         return geomType match {
 
-            case POINT => // case on the length of the point_list, then map
+            case POINT =>
+                if (point_lists.length == 1) {
+                    Point(point_lists.head.head)
+                } else {
+                    MultiPoint(point_lists.map(
+                        (pt: List[(Double, Double)]) => Point(pt.head)))
+                }
 
-            case LINESTRING => null // NYI
+            case LINESTRING =>
+                if (point_lists.length == 1) {
+                    Line(point_lists.head)
+                } else {
+                    MultiLine(point_lists.map(
+                        (ln: List[(Double, Double)]) => Line(ln)))
+                }
 
-            case POLYGON => null // NYI
```

Comment:
Scala has a neat feature for this case by providing function `???` that is of type `Nothing`, therefore acceptable in all places (since `Nothing` is a subtype of every type) and it will throw `NotImplementedException` if ever hit during runtime.